cardano-cli/src/Cardano/CLI/Types/Common.hs

```diff
@@ -398,11 +399,7 @@ type ScriptRedeemerOrFile = ScriptDataOrFile
 -- the script witness data representation.
 --
 -- It is era-independent, but witness context-dependent.
---
--- TODO: Potentially update to WitnessFiles so we can get
--- rid of Maybe (ScriptWitnessFiles). This will be clearer
--- in conveying that we either expect a script witness
--- or a key witness is provided at the signing stage.
+-- NB: This is in the process of being deprecated
```

Comment:
I'm not a fan of vague comments. It will be meaningless for anyone not familiar with the work.
 Can you add a sentence what the replacement should/will look like?